source/guide/3.2/getting-started-prerequisites.md

```diff
@@ -52,15 +53,15 @@ As a general recommendation for the average system, Cloudify would require at le
 
 ### Network
 
-The Manager must be accessible via the following ports:
+The Manager listens on the following ports:
+
+* port 80 - REST API and UI. This port should be accessible when SSL is not enabled.
+* port 443 - REST API and UI. This port should be accessible when SSL is enabled.
+* port 8101 - REST API. This port is used for internal access and as such should only be accessible by [Agent]({{page.terminology_link}}#agent) VMs.
+* port 22 - During [Bootstrapping]({{page.terminology_link}}#bootstrapping), manager components are installed and configured via SSH. It (SSH) is used during [Manager Recovery]({{page.manager_blueprints_openstack_link}}#recovery) as well.
```

Comment:
"It (SSH)" - i'd choose either the one or the other, its weird writing the short version of something only to then clarify using the longer one ;p